packages/storybook/config/storybook/manager-head.html

```diff
@@ -88,4 +88,12 @@
         font-size: 14px;
         margin-bottom: 4px;
     }
+
+    /* Theme dropdown overlay in storybook */
+    div.css-16n0kwd {
```

Comment:
Unfortunately, this is the only way to hook onto the storybook overlay popup.

I think they're based off storybook generated classnames, so they'll only need to be updated if in a later storybook release, they change the original classname.